lib/chargebee/telemetry/request_telemetry_context.rb

```
@@ -0,0 +1,29 @@
+# For more information on how to make changes to this file, please see the README.
+# Reach out to dx@chargebee.com for any questions.
+# Copyright 2026 Chargebee Inc.
+
+module ChargeBee
+  module Telemetry
+    # Context passed to TelemetryAdapter#on_request_start.
+    class RequestTelemetryContext
+      attr_reader :span_name, :resource, :operation, :http_method, :http_url, :server_address,
+                  :chargebee_site, :chargebee_api_version, :sdk_name, :sdk_version, :start_attributes
+
+      def initialize(span_name:, resource:, operation:, http_method:, http_url:, server_address:,
```

RuboCop (1.88.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.0 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)